Makefile

```diff
@@ -25,20 +25,20 @@ unregister-full:
 
 # only aware of aula sources
 sensei: .phony aula.unregister
-	$(EXEC) sensei $(AULA_SOURCES) tests/Spec.hs $(SENSEI_DEFAULT_ARGS) $(SENSEI_ARGS)
+	$(EXEC) sensei -j5 $(AULA_SOURCES) tests/Spec.hs $(SENSEI_DEFAULT_ARGS) $(SENSEI_ARGS)
```

Comment:
I don't want include it in the makefile. I was searching for -j and tried manually with sensei, but sensei cli --help, its documentation, and its source code is not really helpful. I would like what does the "-j5" meam?